### DISCUSSION TITLE
Connect other tools to firefly database

### DISCUSSION THREAD

**Discussion by hopsimcoe** [firefly-iii/firefly-iii#9143] [Firefly III]
Hi everyone,

I'm currently trying to connect other BI tools to the firefly database. For installation of firefly I used Docker compose with the integrated mysql database. On the same server I started another docker container with the BI tool metabase, but I am not able to reach the database from firefly. I get the following error message: 
Could not connect to address=(host=db)(port=3306)(type=master) : Socket fail to connect to host:db, port:3306. db

hostname, port and everything else is the same as it is in the env-files.

Do you guys have any idea what I am doing wrong?

Thank you in advance :)

BR

**Comment by JC5** [comment]
This is a docker thing. I'm pretty sure you can't use "db" but have to use the full name, like "firefly-db-1" or something.
